frontend/src/app/utils/hooks.tsx

```diff
@@ -39,4 +42,6 @@ const useDocumentTitle = (title: string, retainOnUnmount = false) => {
   }, [retainOnUnmount, title]);
 };
 
-export { useOutsideClick, useDocumentTitle };
+const useTypedSelector: TypedUseSelectorHook<RootState> = useSelector;
```

Comment:
Suggestion to move this hook  to the store.ts file so all related store logic is centralized. It could also be renamed to useAppSelector so we follow the[ react-redux guide](https://redux.js.org/usage/usage-with-typescript#typing-the-useselector-hook).

Reply:
Yeah that makes sense to move it to store.ts. For the name of it I originally wrote it as `useAppSelector` like the guide mentions, but wasn't sure whether the name was descriptive enough. I'll update it back since the guide seems to have that as a specific naming convention for `useAppSelector` and `useAppDispatch`